app/controllers/app.users.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchUsers } = require("../models/app.users.model");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((users) => {
+    res.status(200).send({ users });
+  });
+};
```

Comment:
Don't forget the catch block 👍

Reply:
Ive added one, but this is a simple get all users request so the only error handling is for the path? This is caught by express. I am planning to do a single path not found test at the end